### ISSUE TITLE
Card de visualizar sessão deve ter as observações

### ISSUE THREAD

**Issue by victorcmoura** [fga-gpp-mds/2018.1-Reabilitacao-Motora#280]
# Card de visualizar sessão deve ter as observações

### Descrição:
Atualmente, o card de sessão possui apenas o seu nome (data da sessão). Além disso, deve-se adicionar as observações da sessão, escritas com fonte menor que o título, no mesmo card.

![image](https://user-images.githubusercontent.com/26290053/41265316-fb7a0542-6dc7-11e8-90f5-595536b6ec26.png)

### Critérios de Aceitação
Critérios de aceitação para a Issue ser aceita.
- [ ] Testes passando.
- [ ] Build passando

### Tarefas
Seção para tarefas de Issues mais complexas. As tarefas devem garantir o perfeito funcionamento da Issue.
- [ ] Adicionar observações da sessão no mesmo card

**Comment by guilhermedelyra** [comment]
O estilo deve ser identico ao da imagem? Digo, a observação há de ter fundo transparente?

**Comment by arthurbdiniz** [comment]
Não, está livre para usar a criatividade, caso precise de ajuda para definir um design amanhã tenho disponibilidade das 10 horas as 12 na aula de EPS. Apareça lá que a gente constrói o design...

**Comment by jlucassr** [comment]
![sem titulo](https://user-images.githubusercontent.com/23347866/41426436-00a66eac-6fda-11e8-9c46-5ae81868c514.png)

Apesar não está com toda informação da observação trago como exemplo o que poderia servir. Caso contrário me digam como poderia aperfeiçoar.

**Comment by jlucassr** [comment]
Palavras do @guilhermesiqueira 
Aí tu se ****, porque tem como não. Ia precisar de uma scrollbar dentro do inputfield, e isso aí é treta. Não convém colocar essa observação aí não. Pra mim, fica mais intuitivo colocar uma observação um texto assim: "Clique aqui para visualizar as observações da sessão"

**Comment by guilhermedelyra** [comment]
Discordo, acho plenamente possivel fazer da forma que está na imagem da issue (ou ao menos similar).
